app/src/main/java/org/fossasia/openevent/app/data/auth/EncryptionServiceImpl.java

```diff
@@ -1,29 +1,36 @@
 package org.fossasia.openevent.app.data.auth;
 
+import android.content.Context;
 import android.os.Build;
 
-import org.fossasia.openevent.app.data.auth.model.Encryption;
 import org.fossasia.openevent.app.utils.EncryptionUtils;
 
 import javax.inject.Inject;
 
 public class EncryptionServiceImpl implements EncryptionService {
 
-    private AuthHolder authHolder;
+    private Context context;
 
     @Inject
-    public EncryptionServiceImpl(AuthHolder authHolder) {
-        this.authHolder = authHolder;
+    public EncryptionServiceImpl(Context context) {
+        this.context = context;
     }
 
     @Override
-    public void setEncryption(Encryption encryption) {
+    public String encrypt(String credential) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
```

Comment:
We are targetting above Jelly Bean, so why this check?

Also, move the encryption utils code here